dockerscripts/healthcheck.go

```diff
@@ -71,14 +72,17 @@ func findEndpoint() (string, error) {
 	scanner.Split(bufio.ScanLines)
 	// MinIO works on TCP and it is supposed to be
 	// the only process listening on a port on any IP address 
-	// (on :::) inside container.
-	// Since MinIO is running as non-root user, we can
+	// (on ::: or 0.0.0.0) inside container.
+	// Since MinIO may run as non-root user, we can
 	// not depend on the PID/Program name column
 	// of netstat output
 	for scanner.Scan() {
 		line := scanner.Text()
-		if strings.Contains(line, tcp) && strings.Contains(line, anyIP) {
-			newLine := strings.Replace(line, anyIP, "127.0.0.1:", 1)
+		if strings.Contains(line, tcp) && (strings.Contains(line, anyIPv6) || strings.Contains(line, anyIPv4)) {
+			newLine := strings.Replace(line, anyIPv4, "127.0.0.1:", 1)
```

Comment:
Is it still `127.0.0.1` even for IPv6

Reply:
Yes it looks little confusing.. can you shed some light @nitisht

Reply:
You're right @kannappanr if it is only IPv6, https://github.com/minio/minio/pull/8046/files#diff-e354ae019f16b5ff940a9e155ae1ee26R84 should use `::1`, will make the change